applications/GeoMechanicsApplication/custom_utilities/README.md

```diff
@@ -58,15 +58,25 @@ $$M = \int_\Omega N_{u}^T \rho N_u d\Omega$$
 
 Where $\Omega$ is the domain, $N_u$ is the displacement shape function and $\rho$ is the density matrix that holds density for all directions.
 
+### Stiffness Matrix (K)
+
+The mathematical definition is:
+$$K = \int_\Omega B^T C_{constitutive} B d\Omega$$
+
+Where $\Omega$ is the domain, $B$ is the B-matrix and $C_{constitutive}$ is the constitutive matrix.
+
+
 ### Damping Matrix (D)
 
 The mathematical definition is:
 $$D = \alpha_R M + \beta_R K$$
 
 Where $M$ and $K$ are the mass and stiffness  matrices respectively and $\alpha_R$ and $\beta_R$ are the coefficients from the Rayleigh Method.
 
-File equation_of_motion_utilities.hpp includes 
+File equation_of_motion_utilities.h includes 
 -  CalculateMassMatrix function
+-  CalculateStiffnessMatrixGPoint provides a stiffness matrix for a specific integration point
+-  CalculateStiffnessMatrix provides a stiffness matrix for all integration points
```

Comment:
Perhaps rephrase this, to emphasize that this function works on element level?
```suggestion
-  CalculateStiffnessMatrix provides a stiffness matrix for an element
```

Reply:
Actually, it was my first option but then I made it like others. ;( Changed.